safe/reportv4/impact_report.py

```diff
@@ -281,6 +281,8 @@ def __init__(
         self._aggregation_impacted = (
             aggregation_impacted or
             self._impact_function.aggregation_impacted)
+        if extra_layers is None:
```

Comment:
Why not:
```
if not extra_layers:
    extra_layers = []
```
Useful if you passed an empty dict, it still gets overwritten.

But still, I think we didn't have a chance to use this extra_layers for now.